packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -23,3 +27,40 @@ export const Default = {
     ],
   },
 } satisfies Story;
+
+export const MoreActions = {
+  args: {
+    columns: ["name", "age", "birthday"],
+    features: [
+      {
+        name: Symbol("More actions"),
+        watch: [],
+        header: {
+          actions: (column) =>
+            column === "name"
+              ? [
+                  {
+                    iconComponent: h(OnyxIconButton, {
+                      label: "More actions",
+                      icon: moreHorizontal,
+                      color: "neutral",
+                    }),
```

Comment:
The `ᐧᐧᐧ` Button should be a default feature, when the conditions are met. So it shouldn't be defined here, but in the `features/index.ts` and be rendered instead of the `iconComponent`s